libs/blocks/modal/modal.css

```diff
@@ -1,5 +1,7 @@
 :root {
   --modal-focus-color: #109cde;
+  --spacing-xl: 56px;
+  --aspect-ratio-tall: 9/16;
```

Comment:
Is this set in stone? Do we foresee use-cases for other values, justifying having this authorable as well?

Reply:
This is the only use-cause I have at the moment. Specifically for tall videos, which do not display well in our current modals.

I'll update the spacing token as mentioned in above comment.

Reply:
Same with this, we already declare this aspect ratio in styles.css. Seem like we shouldn't need this var